src/main/resources/edu/cornell/kfs/fp/cu-ojb-fp.xml

```diff
@@ -733,8 +733,53 @@
     </reference-descriptor>
 </class-descriptor>
 
-</descriptor-repository>
 
+  <class-descriptor class="edu.cornell.kfs.fp.businessobject.IncomingWireAchMapping" table="FP_INCOMING_WIRE_ACH_MAP_TC">
+    <field-descriptor name="shortDescription" column="SHORT_DESC" jdbc-type="VARCHAR" primarykey="true" index="true" />
```

Comment:
Is the "shortDescription" going to be unique enough for PK use? Will users or automated processes end up having trouble fitting an appropriate description into this column that doesn't clash with others?